src/Frontend/Page/User.hs

```diff
@@ -39,31 +39,43 @@ import qualified Text.Digestive.Lucid.Html5 as DF
 data PageUserSettings = PageUserSettings User
   deriving (Eq, Show, Read)
 
-instance Page PageUserSettings
+instance Page PageUserSettings where
+    -- The use settings page always goes to the profile of the current logged in user.
```

Comment:
does that mean you rely on the URL to not contain the uid?  because i think that's changed, and even if not it might change in the future.